fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -482,6 +484,13 @@
     @VariableMgr.VarAttr(name = SINGLE_NODE_EXEC_PLAN, flag = VariableMgr.INVISIBLE)
     private boolean singleNodeExecPlan = false;
 
+    @VariableMgr.VarAttr(name = ENABLE_DESC_TBL_CACHE)
+    private boolean enableDescTblCache = false;
```

Comment:
default false? Does user needs to understand this parameter?

Reply:
I think user don't